gremlin-shaded/pom.xml

```diff
@@ -24,6 +24,67 @@ limitations under the License.
     </parent>
     <artifactId>gremlin-shaded</artifactId>
     <name>Apache TinkerPop :: Gremlin Shaded</name>
+    <packaging>bundle</packaging>
+    <properties>
+        <!--
+        As this jar is shaded, the maven-bundle-plugin is not able to process exported and imported packages from generated classes.
+        So we must explicitely tell which packages are exported and imported by this bundle to the maven-bundle-plugin.
+        -->
+        <osgi.import>
```

Comment:
how do we know what should be added to `<osgi.import>` and `<osgi.export>`?  this goes back to my earlier question of how to "maintain" this stuff over time?